packages/client/components/EditingStatus/EditingStatusText.tsx

```diff
@@ -27,6 +27,11 @@ const useTimeFrom = (timestamp: string) => {
       window.clearTimeout(timeoutRef.current)
     }
   }, [makeTimeFrom, timeFrom, timestamp])
+  // to update on toggle
+  useEffect(() => {
+    setTimeFrom(makeTimeFrom())
+  }, [timestamp])
```

Comment:
+1 correct would be to add `makeTimeFrom` a dependency here as well. Because `makeTimeFrom` also depends on `timestamp` it's not a problem right now, but could theoretically be one in the future.